cheatsheets_draft/GraphQL_Cheat_Sheet.md

```diff
@@ -0,0 +1,257 @@
+# GraphQL Cheat Sheet
+
+## Introduction
+
+Below are some quick high-level ideas to keep in mind when building a secure API with GraphQL.
+
+- Strict input validation is highly recommended and easy
+- Expensive queries can easily lead to Denial of Service (DoS); there are several defenses ranging from simple to complex
+- It is very important to implement proper access control (authorization) but it can be tricky
+- Some default configurations (Introspection, GraphiQL) should be disabled/changed before releasing an API to production
+
+## Common Attacks
+
+- [Injection](https://github.com/OWASP/API-Security/blob/master/2019/en/src/0xa8-injection.md) - this usually includes but is not limited to:
+    - [SQL](https://owasp.org/www-community/attacks/SQL_Injection) and [NoSQL](https://www.netsparker.com/blog/web-security/what-is-nosql-injection/) injection
+    - [OS Command injection](https://owasp.org/www-community/attacks/Command_Injection)
+    - [SSRF](https://portswigger.net/web-security/ssrf) and [CRLF](https://owasp.org/www-community/vulnerabilities/CRLF_Injection) [injection](https://www.acunetix.com/websitesecurity/crlf-injection/)/[Request](https://portswigger.net/web-security/request-smuggling) [Smuggling](https://www.pentestpartners.com/security-blog/http-request-smuggling-a-how-to/)
+- [DoS](https://owasp.org/www-community/attacks/Denial_of_Service) ([Denial of Service](https://www.cloudflare.com/learning/ddos/glossary/denial-of-service/))
+- [IDOR](https://portswigger.net/web-security/access-control/idor)
```

Comment:
How about moving IDOR to Broken Authorization section or removing? I think IDOR issues are pretty much covered by BOLA (Broken Object Level Authorization) you mentioned below 😃

Reply:
This section is basically just a condensed version of the major sections below. I'll change this to reflect whatever changes are made in the major sections.